### PR TITLE
Update Mission Control to Version 3.3.0

### PR DESCRIPTION
**Pull request by amithins**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] Changelog.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**: Updates Mission Control version to 3.3.0

### CODE REVIEW

File: stable/mission-control/values.yaml [hunk 1]
```diff
@@ -118,27 +120,14 @@ ingress:
...
-
 missionControl:
   replicaCount: 1
   name: mission-control
   appName: jfmc-server
   home: /var/opt/jfrog/mission-control
   image: docker.bintray.io/jfrog/mission-control
   ## Note that by default we use appVersion to get image tag
-  # version: 3.2.0
+  # version: 3.3.0
```
Comment (eldada) [inline stable/mission-control/values.yaml]: Please remove the `3.3.0`. Leave is just `# version:`. Easier to support later on. Do it for the other services too.
We should have the app version written in the `Chart.yaml` only.

File: stable/mission-control/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,10 @@
 # JFrog Mission-Control Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [0.7.0] - Nov 16, 2018
+* Updated Mission-Control version to 3.3.0
+* Remove usage of certificates for internal communication
+
```
Comment (eldada) [inline stable/mission-control/CHANGELOG.md]: Please add upgrade instructions from 3.2.0 (if needed)

Comment (amithins) [reply stable/mission-control/CHANGELOG.md]: No extra steps needed for this upgrade.

File: stable/mission-control/templates/insight-server-deployment.yaml [hunk 3]
```diff
@@ -104,33 +104,18 @@ spec:
...
-        - name: JFMC_MISSION_CONTROL_CERT
-          value: "{{ .Values.insightServer.home }}/{{ .Values.insightServer.name }}/_MASTER_/data/contexts/security/jfmc.crt"
-        - name: JFMC_INSIGHT_SERVER_CERT
-          value: "{{ .Values.insightServer.home }}/{{ .Values.insightServer.name }}/_MASTER_/data/contexts/security/insight.crt"
-        - name: JFMC_INSIGHT_SERVER_KEY
-          value: "{{ .Values.insightServer.home }}/{{ .Values.insightServer.name }}/_MASTER_/data/contexts/security/insight.key"
+        - name: JFMC_INSIGHT_SERVER_DB
+          value: "{{ .Values.mongodb.db.insightServerDb }}"
+        - name: JFIS_ALLOW_IP
+          value: "0.0.0.0/0"
```
Comment (eldada) [inline stable/mission-control/templates/insight-server-deployment.yaml]: @amithins - please make this a configurable parameter. Can help with simple hardening.

Comment (rimusz) [reply stable/mission-control/templates/insight-server-deployment.yaml]: @amithins can you update the PR as per @eldada request, so we can them merge the PR, thanks

**Comment by rimusz** [comment]
LGTM

**Pull request merged**